Example/ExampleTests/WeatherViewControllerTests.swift

```diff
@@ -11,67 +11,78 @@ import YumemiWeather
 @testable import Example
 
 class WeatherViewControllerTests: XCTestCase {
-
+    
     var weahterViewController: WeatherViewController!
     var weahterModel: WeatherModelMock!
+    var disasterModel: DisasterModel!
     
     override func setUpWithError() throws {
         weahterModel = WeatherModelMock()
+        disasterModel = DisasterModelImpl()
         weahterViewController = R.storyboard.weather.instantiateInitialViewController()!
         weahterViewController.weatherModel = weahterModel
+        weahterViewController.disasterModel = disasterModel
         _ = weahterViewController.view
     }
-
+    
     override func tearDownWithError() throws {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
-
+    
     func test_天気予報がsunnyだったらImageViewのImageにsunnyが設定されること_TintColorがredに設定されること() throws {
         weahterModel.fetchWeatherImpl = { _ in
             Response(weather: .sunny, maxTemp: 0, minTemp: 0, date: Date())
         }
         
-        weahterViewController.loadWeather()
-        XCTAssertEqual(weahterViewController.weatherImageView.tintColor, R.color.red())
-        XCTAssertEqual(weahterViewController.weatherImageView.image, R.image.sunny())
+        weahterViewController.loadWeather(nil)
+        DispatchQueue.main.async {
+            XCTAssertEqual(self.weahterViewController.weatherImageView.tintColor, R.color.red())
+            XCTAssertEqual(self.weahterViewController.weatherImageView.image, R.image.sunny())
```

Comment:
![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg)

ここの値を変えてテストを実行するとどうなるでしょうか？

Reply:
値を変えてテストするというのは具体的にどういうことでしょうか？
メソッド名とテストの内容が一致しているのでどの部分を修正して良いかわからないため、教えていただきたいです。🙇‍♂️

Reply:
例えばですが、以下に書き換えてテストがパスしてしまうことを確認してみてください。

```suggestion
            XCTAssertEqual(self.weahterViewController.weatherImageView.tintColor, R.color.blue())
            XCTAssertEqual(self.weahterViewController.weatherImageView.image, R.image.rainy())
```

Reply:
通ってしまいますね、、、
原因調べてみます。

Reply:
非同期のデータ取得のテストが正しくできていないことがわかりました。
非同期であっても正しくテストができるように修正しました。